### ISSUE TITLE
[Nodo] Control de acceso

### ISSUE THREAD

**Issue by darkjavi** [makespacemadrid/Jarvis#35]
**Propósito del nodo:**
  -Controlar la cerradura de la puerta interior del espacio.
  -Botón para activar un timbre para que los visitantes puedan llamar.
  -Lector nfc

**Detalles de implementación:**
-Personalmente creo que es mejor utilizar una cerradura de las que abren cuando aplicas corriente, mejor que el modulo de cierre magnético que tenemos. De ese modulo lo que si podríamos usar es el sensor de cierre para saber cuando se ha cerrado la puerta

**Estado**:
HW: 80%
SW: 50%

Numero de nodos necesarios: 2 , Construidos 2.

**Comment by trastejantEdu** [comment]
¿Novedades en este tema? propongo añadir la tarjeta con la que abrimos el cierre como opción complementaria al pin-pad. Es fácil de implementar y es única, a diferencia del pin, lo que facilita el control de acceso.

**Comment by darkjavi** [comment]
Manuel esta trabajando en este modulo, en una primera versión que este conectada a la apertura eléctrica de la puerta que hemos instalado en el ultimo mes.

Para la V2 un pin pad, y para la V3 habria que ver si se puede integrar esto (http://hackaday.com/2016/04/13/carontepass-open-access-control-for-your-hackerspace/) en el firmware de Jarvis

**Comment by trastejantEdu** [comment]
Exacto, eso es justo lo que tenia en mente. Ahora precisamente estoy cacharreando con esos módulos para un robot que estoy diseñando, así que puedo aportar en ello.

**Comment by darkjavi** [comment]
En esta semana he estado trabajando en el nfc del espacio.
He actualizado a versión 2 el abre puertas exterior(el de la persiana), de momento no está conectado a Jarvis.

Habia un par de rc522 por ahí sueltos y he instalado uno en el modulo abre puerta interior(este si es Jarvis) para probarlo, de momento tirando del mismo array de autorización que la puerta exterior.
